cfy_cluster_setup/config_files_templates/manager_config.yaml

```diff
@@ -21,17 +21,24 @@ rabbitmq:
   ca_path: {{ ca_path }}
 
 postgresql_server:
-  ssl_enabled: true
-  ca_path: {{ ca_path }}
-
-  postgres_password: {{ creds.postgresql.postgres_password }}
   cluster:
     nodes: {{ postgresql_cluster }}
+  {% if external_db_configuration %}
+  ca_path: {{ external_db_configuration.ca_path }}
+  {% else %}
+  ca_path: {{ ca_path }}
+  ssl_enabled: true
+  postgres_password: {{ creds.postgresql.postgres_password }}
+  {% endif %}
 
 postgresql_client:
+  {% if external_db_configuration %}
```

Comment:
I think you might want some minuses (-%} or {%-) to stop getting ugly extra whitespace with this.

Reply:
I don't think I fully understand this comment. Isn't this the Jinja template syntax `{% parameter %}`?

Reply:
Sure, but you can also add - next to the % signs to suppress some whitespace. As written you will get something like:
somestuff

postgresql_client:
  
  say hello
  

otherstuff

Reply:
...though with indentation. Thanks, github.

Reply:
With a -%} on the first if you will get something like:
somestuff

postgresql_client:
  say hello
  

otherstuff

Reply:
Github's ui has also eaten the extra newlines
I really detest this editor...

Reply:
Anyway, I suggest trying rendering it by hand to try to get to the point that you don't have:
`somestuff

postgresql_client:
  say hello
  

otherstuff`

Reply:
...even when I use the insert code feature? Really?
Ok, so I suggest rendering it by hand to try to get to the point that it looks nice and cleanly written.

Reply:
Yep, I tried it now on a dummy example and the result was that all the blank lines were deleted.
 
```
rabbitmq:
  ca_path:
  cluster_members:

{%-if '1' in node.hostname-%}
networks:
  load_balancer: {{ node.hostname }}
{%-endif-%}

postgresql_server:
  ssl_enabled: true
  ca_path:
```

turned out to: 
```
rabbitmq:
  ca_path:
  cluster_members:networks:
  load_balancer: yoni-1postgresql_server:
  ssl_enabled: true
  ca_path:
```

Reply:
Looks good!